### ISSUE TITLE
Add UI element to enable/disable using the IMU

### ISSUE THREAD

**Issue by lwillmeth** [LBCC-SpaceClub/HAB2017#8]
We should be able to manually set the direction of the ground station instead of using the IMU.

Maybe a dropdown with IMU, North, South, East, West?

**Comment by proutyio** [comment]
Is this in addition to entering the values and manually setting it?

**Comment by lwillmeth** [comment]
Yes.  Right now we can manually set the location of both units, but we also need to be able to set the orientation of the tracker.  ServoControl/StepperControl will check that GUI field and if it's set, use that value instead of the magnetometer.
